src/Common/MoistThermodynamics/states.jl

```diff
@@ -276,8 +298,10 @@ function LiquidIcePotTempSHumNonEquil(
     ρ::FT,
     q_pt::PhasePartition{FT},
     maxiter::Int = 5,
-    tol::FT = FT(1e-1),
+    temperature_tol::FT = FT(1e-3),
```

Comment:
I don't know why we need this in non-equilibrium (by definition, saturation adjustment does not apply). If we do for some reason, it should be 
```suggestion
    potential_temperature_tol::FT = FT(1e-2),
```
(or 1e-1).

Reply:
I found by trial and error that a default value of at least 1 helps. @tapios would 1 be too large as a default?

Reply:
@smarras79 We changed what the convergence criterion means. In DYCOMS, `1e-1` should work fine with 1 iteration, from the tests we looked at.